app/stats/knights.py

```diff
@@ -0,0 +1,64 @@
+from typing import Optional
+from app.stats.armor import Armour
+from app.stats.weapon import Weapon
+from app.stats.potion import Potion
+
+
+class Knights:
+    knights = {}
```

Comment:
Why you need to save all instances inside class?

Reply:
this is needed to assign variables to the names of knights in main.py in lines 96-99

Reply:
But you can return it from your method. No need to save it into class